server/docker/common-services.yml

```diff
@@ -14,7 +14,7 @@ services:
       - ./../app/:/app/:ro
       - ./../app/:/etc/extender/apps/:ro
       - ./../configs:/etc/defold/extender:ro
-      - ${DYNAMO_HOME:-dummy:/dynamo_home}
```

Comment:
In such configuration if DYNAMO_HOME is not set - it creates a emptyr folder `dummy` and mount it.